### PR TITLE
:sparkles: Home Icon

### PR DESCRIPTION
**Pull request by J-Bannach**
- Add Home Icon with link
- Add story for storybook

### CODE REVIEW

File: src/app/components/HomeLink/HomeLink.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  link: string;
+};
+
+function HomeLink({ icon, link }: HomeLinkProps): JSX.Element {
+  return (
+    <Link to={link} className={styles.homeLink}>
+      {icon}
+    </Link>
+  );
+}
```
Comment (lmachens) [inline src/app/components/HomeLink/HomeLink.tsx:16]: If this is a `HomeLink`, shouldn't it always link to the homepage like `/`?
If you like to use this link for other routes too, I would rename it to something like `IconLink`.

File: src/app/components/HomeLink/HomeLink.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import HomeIcon from '../Icons/HomeIcon';
+
+export default {
+  title: 'Component/Link',
+  component: HomeIcon,
+};
+
+export const Home = (): JSX.Element => <HomeIcon></HomeIcon>;
```
Comment (lmachens) [inline src/app/components/HomeLink/HomeLink.stories.tsx:9]: This is a story about your HomeLink component (`HomeLink.stories.tsx`), but you display the `HomeIcon`.

**Review by lmachens — COMMENTED** [review]
gj

File: src/app/components/HomeLink/HomeLink.tsx (hunk 1 above)
Comment (J-Bannach) [reply src/app/components/HomeLink/HomeLink.tsx]: Changed the link target to `/`, thanks.

**Pull request merged**